### ISSUE TITLE
Problem with .id

### ISSUE THREAD

**Issue by 92Eli** [92Eli/Which-Way#1]
If you play through the CTH story once and then select the game to do it again, it will be stuck.
Usually after you do it again then you will get put on the id=5 page ("What???"...).
It only works if you restart the program.
Line 204 is where id:5 is defined.

**Comment by gsquaredxc** [comment]
I don’t think this is a problem with the id:5 itself, I think it may have to do with something else in the program. Perhaps the timer should be displayed for debugging and see if the timer has anything to do with it.

**Comment by 92Eli** [comment]
Maybe I should use the number the scenario comes in the array (like cth[0]) instead of .id (like cth[0].id, which would be 0 anyways.
